keyboards/handwired/tractyl_manuform/5x6_right/rpi2040/readme.md

```diff
@@ -0,0 +1,108 @@
+# Schievel's RP2040 Tractyl Manuform (5x6) with a right side trackball
+
+![Imgur](https://i.imgur.com/Xvua5XK.jpeg)
+
+* Split Hand Pin, using GP28 on left side
+* Full Duplex Serial/USART using GP1 and GP2. Do not swap pins on other side, connect GP1 left to GP1 right.
+* PWM WS2812 RGB using GP21
+* PMW3360 sensor on SPID0, with GP17 as CS pin
+* Encoder using GP26 and GP27
+* SSD1306 OLED display (128x64) using GP3 and GP4 on I2CD1
+* Reset LED on GP25 (blue LED an many boards)
+* Double tap onboard reset switch (many boards have that) to enter bootloader
+* Connect GP22 to ground 2 times in RESET_PIN_DOUBLE_TAP_RESET_TIMEOUT milliseconds (default 1000ms) to enter bootloader
+
+Ciruit diagram:
```

Comment:
Could we please not? I am very proud of this ascii circuit diagram :) 

Also it would have helped me tremendously when there was a circuit diagram available when I started out with this. I know once you know how this works it seems kind of obvious, but in the beginning this was a very frustrating guessing game.

Reply:
Well, you have most of this content in a linked repo. So not sure how much of it really needs to be here.